src/components/Create/BudgetForm.tsx

```diff
@@ -101,11 +124,18 @@ export default function BudgetForm({
           />
         )}
 
-        {showFundingFields && (
+        {showFundingFields && target === '0' && (
```

Comment:
this was originally conditional for when target == 0, but generally i've started leaning more towards showing things unconditionally. my assumption is that it's more helpful for folks to understand how the target works, even if they aren't setting their target to 0. wdyt @aeolianeth ?